packages/web-components/src/tooltip/tooltip.template.ts

```diff
@@ -0,0 +1,12 @@
+import { html } from '@microsoft/fast-element';
+import type { Tooltip } from './tooltip.js';
+
+/**
+ * Template for the tooltip component
+ * @public
+ */
+export const template = html<Tooltip>`
+  <template id="tooltip-${x => x.anchor}" popover aria-hidden="true">
```

Comment:
Does this get properly overridden if someone wants to assign a specific id attribute?

Reply:
Updated to use an `id` attr. 

```
<template id="${x => x.id}" popover aria-hidden="true">
    <slot></slot>
  </template>
  ```

Reply:
@davatron5000 id will reflect, so this shouldn't need to be bound on the template - it'll happen automagically.

Reply:
D'oh. Removed.